core/templates/dev/head/pages/exploration-editor-page/editor-tab/exploration-editor-tab.directive.ts

```diff
@@ -191,6 +191,11 @@ oppia.directive('explorationEditorTab', ['UrlInterpolationService', function(
             angular.copy(displayedValue));
         };
 
+        ctrl.saveSolicitAnswerDetails = function(displayedValue) {
+          ExplorationStatesService.saveSolicitAnswerDetails(
+            ctrl.stateName, angular.copy(displayedValue));
```

Comment:
Do we have to call `StateEditorService.set...` here as well? (See line 182, 190 etc.)

Reply:
Done, added.